bc_obps/registration/migrations/0115_fix_typo_in_regulated_product_name.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.0.14 on 2025-05-26 19:11
+
+from django.db import migrations
+
+
+def fix_typo(apps, schema_editor):
+    RegulatedProduct = apps.get_model('registration', 'RegulatedProduct')
+    RegulatedProduct.objects.filter(name='Compression, positve displacement - consumed energy').update(
+        name='Compression, positive displacement - consumed energy'
+    )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('registration', '0114_V3_2_1'),
+    ]
+
+    operations = [
+        migrations.RunPython(fix_typo),
```

Comment:
In reg we often include reverse_code=migrations.RunPython.noop and elidable=True with these kinds of forward-only migrations, e.g. /home/briannacerkiewicz/cas-registration/bc_obps/registration/migrations/0095_populate_operation_timeline_with_prod_data.py